### ISSUE TITLE
Opening modal from tabs - faded animation bug

### ISSUE THREAD

**Issue by antoine92190** [ionic-team/ionic#11235]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

On iOS device, I open a modal from tabs as explained in the docs: (ionSelect)="openModal()".

It works fine but when I implement a navigation structure inside the modal, then iOS sliding animation is buggy.

When I use nav.push() inside the modal, the faded background during the animation is the background of the previous selected tab and not the actual one.
If I navigate further inside the modal, the faded background is the one of the previous page, and not the actual one.

**Expected behavior:**

Navigation inside modal from tabs should have animation consistencies.

**Steps to reproduce:**

You can have a look at the gif below. I added some colors to make the issue more obvious.
- You can see that in the "New Event" Page,  when I click on the "Invite" button, the faded background is actually the one of my user profile (the tab selected before).
- In the "Invite" Page, when I click on the "Following" button, I can see a faded background with red lists, while lists should be green.
- Same behaviour happens when clicking on the back button

![modal](https://cloud.githubusercontent.com/assets/16256207/25071038/d7f8ed7e-22ad-11e7-96f3-553d87864969.gif)

**Related code:**

```
insert any relevant code here
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.4.0 
Ionic Framework Version: 3.0.1
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.3.1
ios-deploy version: 1.9.0 
ios-sim version: 5.0.13 
OS: macOS Sierra
Node Version: v6.10.0
Xcode version: Xcode 8.3.1 Build version 8E1000a
```

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us. Would you mind posting a minimal repo we could use to reproduce this issue?

**Comment by antoine92190** [comment]
You can try this repo on Google Chrome using an iPhone simulator:
- git clone https://github.com/antoine92190/ionic-modal.git
- npm install

You can see that the Home tab has a red background.
If you click on the About tab, a modal is opened.
Then click on the "Go to Page1" button
There you can see a faded red background during the iOS animation.

**Comment by lpsantana** [comment]
Same occurring here. Any news on this?

**Comment by antoine92190** [comment]
@lpsantana As a workaround, you can add the below code in the page you opened as a modal:
 
In the constructor:
`this.pageElement = document.querySelector('.ion-page');`

Then create a function:
`  ionViewDidEnter() {
      this.tabBarElement.style.display = 'none';
  }`

And when you want to close the modal, use:
`this.tabBarElement.style.display = 'initial';`

**Comment by antoine92190** [comment]
Fixed by #12747
